backend/src/image/ImageService.ts

```diff
@@ -0,0 +1,41 @@
+import fs, { PathLike } from 'fs';
+import path from 'path';
+import _ from 'lodash';
+import { Files } from 'formidable';
+import config from 'config';
+import { Service } from 'typedi';
+import BlogError from '@src/common/error/BlogError';
+import ImageRepository from '@src/image/ImageRepository';
+import { ImageDoc } from '@src/image/Image';
+import { UploadImageParamDto } from './ImageDto';
+import { BlogErrorCode } from '../common/error/BlogErrorCode';
+
+@Service()
+export default class ImageService {
+  public constructor(private readonly _imageRepository: ImageRepository) {
+  }
+
+  public uploadImage = (paramDto: UploadImageParamDto) => {
+    const { files } = paramDto;
```

Comment:
Add a validation. If there is a file with same file name with the given file name, notify the duplication by a `BlogError`.